rct_optimizations/src/rct_optimizations/dh_chain_kinematic_calibration.cpp

```diff
@@ -416,6 +480,9 @@ KinematicCalibrationResult optimize(const KinematicCalibrationProblemPose6D &par
     problem.AddResidualBlock(cost_block, nullptr, target_chain_dh_offsets.data());
   }
 
+  // Print optimization parameter labels
+  printOptimizationLabels(problem, param_names, param_labels, param_masks);
+
```

Comment:
Is this really needed or is this for debug?

Reply:
Just for debug.